### PR TITLE
SciPy Ar 2013

### PR DESCRIPTION
**Pull request by akielbowicz**
title: SciPy Ar 2013
dir: scipy-ar-2013
youtube_list:
https://www.youtube.com/playlist?list=PLIFfMgsU85R_ZhNEadTCxedmRSXEDF56W
language: spa
dates:
begin: 2013-05-16
end: 2013-05-18
schedule: http://scipyla.org/conf/2013/schedule.html

### CODE REVIEW

File: scipy-ar-2013/category.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "title": "SciPy Ar 2013",
```
Comment (redapple) [inline scipy-ar-2013/category.json]: I wonder if this should not go next to the existing `scipy-la-2016`. What do you think @akielbowicz

File: scipy-ar-2013/videos/scipyconar-2013-inteligencia-artificial-con-python-pedro-fisanotti-dia-1-de-3.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "",
```
Comment (redapple) [inline scipy-ar-2013/videos/scipyconar-2013-inteligencia-artificial-con-python-pedro-fisanotti-dia-1-de-3.json]: I wonder if we cannot use the data from http://scipyla.org/conf/2013/activities/index.html
```
Día Uno:
(T) ¿Por qué Inteligencia Artificial?
Ejemplos de problemas para despertar el interés en el tema.
Elección de un problema particular para aplicar todo lo que se vaya viendo a lo largo del taller.
Introducción al concepto de Inteligencia Artificial, enfoques posibles, rama elegida para el taller.
(T) Agentes racionales, tipos de ambientes, limitaciones.
(T) Noción básica de lo que es un problema de búsqueda. Diferencia entre buscar un estado y buscar un camino de acciones.
(T+P) Formulación de problemas de búsqueda (elementos que debemos entender y definir de nuestro problema).
Representación del estado.
Definición de las acciones posibles desde cada estado.
Función de costo.
Función de comprobación de meta o función de valoración.
Función heurística.
(T) Algoritmos para resolver problemas de búsqueda. Idea general y diferentes tipos.
(T) Presentación de la biblioteca a usar en la práctica: SimpleAI. Ventajas de usar python para este tipo de programación.
(P) Instalación de las herramientas necesarias en las pcs de los asistentes.
```

File: scipy-ar-2013/videos/scipyconar-2013-python-cientifico-episodio-i-damian-avila-dia-1-de-3.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "",
```
Comment (redapple) [inline scipy-ar-2013/videos/scipyconar-2013-python-cientifico-episodio-i-damian-avila-dia-1-de-3.json]: same as for Juan Pedro Fisanotti's 3 days, we can probably put text from
http://scipyla.org/conf/2013/activities/index.html > Python Científico: Episodio I - Damián Avila

**Comment by redapple** [comment]
GitHub is reporting conflicts preventing merging.

**Comment by akielbowicz** [comment]
Fix merge on #801

**Pull request closed without merging**